TGEMO.OWL

```diff
@@ -872,64 +872,27 @@
     
 
 
-    <!-- http://gemma.msl.ubc.ca/ont/TGEMO#OVE26 -->
+    <!-- http://gemma.msl.ubc.ca/ont/TGEMO_00173 -->
```

Comment:
The last ID @ppavlidis generated was `TGEMO:00171`, this term can safely use `TGEMO_00172`.